db/seedData.js

```diff
@@ -0,0 +1,39 @@
+const titleData = require('./data/titleData');
+const mapData = require('./data/mapData');
+const motherData = require('./data/MomaDummyData');
+const { Client } = require('pg');
+
+const client = new Client({
+  database: 'restaurantyelp',
+});
+
+client.connect();
+
+Object.keys(motherData.types).forEach((type) => {
+  const queryString = 'insert into Types (type) values ($1)';
+  client.query(queryString, [type], (err) => {
+    if (err) throw new Error(err.stack);
+  });
+});
+
+const insertData = (title, map) => {
```

Comment:
Do you have tests for these functions?

Reply:
@MichaelDurfey @ZeMunchkin those are integration tests, then, not unit tests.

Reply:
But it is the seed file, not a feature, so I think okay in this case.